### PR TITLE
XHTML Compliance - Diagnostics Tables

### PR DESCRIPTION
**Pull request by N0YB**
html id's not permitted to begin with a number.
html id's not permitted to contain '/'
add prefix (entry_) and replace slash with hyphen.
table entry id format becomes: entry_<ip address>-<cidr>
replacing the format: <ip address>/<cidr>
does not change the displayed format.

### CODE REVIEW

File: usr/local/www/diag_tables.php [hunk 1]
```diff
@@ -147,7 +147,7 @@ function del_entry(entry) {
 	foreach ($entries as $entryA):
 		$entry = trim($entryA);
 ?>
-	<tr id="<?=$entry?>">
+	<tr id="entry_<?=str_replace("/", "-", $entry)?>">
```
Comment (rbgarga) [inline usr/local/www/diag_tables.php]: Missing semicolon

**Comment by rbgarga** [comment]
It breaks javascript function that fade out the item when you delete it

**Comment by rbgarga** [comment]
Something wrong happened here, for some reason commit bf2dba7 is missing from Pull Request, if you check https://github.com/pfsense/pfsense/pull/1719/files you will see it only shows 7d8dd0b

**Comment by N0YB** [comment]
Re-pushed and it seem to be there now.  Please let me know if it's not.  Thanks.

**Pull request merged**